.yamato/all-urp_performance_lostcrypt.yml

```diff
@@ -33,6 +33,8 @@ URP_Performance_LostCrypt_trunk:
          rerun: on_new_revision
       -  path: .yamato/urp_performance_lostcrypt-win-dx12.yml#URP_Performance_LostCrypt_Win_DX12_performance_build_editmode_mono_Linear_trunk
          rerun: on_new_revision
+      -  path: .yamato/urp_performance_lostcrypt-osx-metal.yml#URP_Performance_LostCrypt_OSX_Metal_performance_build_editmode_mono_Linear_trunk
```

Comment:
Same here as for the boat attack comment: is build_editmode the only perf test to add to the nightly?

Reply:
yes, the others fail - fogbugz linked in description

Reply:
I think @theo-pnv is right here, no? The crashes only surfaced for boat attack, my bad! I'll fix it, thanks for spotting!

Reply:
oh thank you, i thought the crash was in both

Reply:
again, great spot. added them in b729680, jobs are running [here](https://unity-ci.cds.internal.unity3d.com/job/9826974) and [here](https://unity-ci.cds.internal.unity3d.com/job/9826966) ✔️

Reply:
reverted with 47dc00a, see slack.